packages/@adobe/spectrum-css-temp/components/tabs/index.css

```diff
@@ -271,3 +268,10 @@ governing permissions and limitations under the License.
   flex-shrink: 0;
   flex-basis: 0%;
 }
+
+.spectrum-TabsPanel-tabpanel {
+  flex-grow: 1;
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+}
```

Comment:
Out of curiosity, did you have a use case that prompted the above css change? I figured that the user could handle the styling of the tabpanel themselves via styling the `Content` they pass

Reply:
you are right the last 3 props are not needed - I had them in there just for convenience. The flex-grow is required although